### PR TITLE
Fixed NodeJs getJwtToken

### PR DESCRIPTION
**Pull request by faiqibrahim**
Fixed getJwtToken Implementation as per docs

### CODE REVIEW

File: language-examples/nodejs/rest/app.js [hunk 1]
```diff
@@ -240,13 +240,10 @@ const port = process.env.PORT || 8080;
 app.listen(port, () => console.log(`Listening on port ${port}...`));
 
 function getJwtToken(auth_url, client_id, client_secret) {
-  const url = `${auth_url}/oauth2/token`;
-  const encoded = Buffer.from(`${client_id}:${client_secret}`).toString(
-    "base64"
-  );
+  const url = auth_url?.endsWith('/oauth2/token') ? auth_url : `${auth_url}/oauth2/token`;
```
Comment (cjcenizal) [inline language-examples/nodejs/rest/app.js:243]: Nit: Since the suffix is repeated and tightly coupled, we can make it a bit easier to read and maintain by extracting it into a variable:

```ts
const OAUTH_SUFFIX = "/oauth2/token";
const url = auth_url?.endsWith(OAUTH_SUFFIX) ? auth_url : `${auth_url}${OAUTH_SUFFIX}`;
```

Comment (cjcenizal) [reply language-examples/nodejs/rest/app.js:243]: Is it possible for `auth_url` to end with `"/oauth2/token/"`? This edge case will result in a bug. Should we sanitize `auth_url` by trimming the last character if it's a trailing slash?

**Review by cjcenizal — APPROVED** [review]
I didn't test locally, but the code looks good. I had one nit and spotted one potential bug.
